components/script/webdriver_handlers.rs

```diff
@@ -356,6 +356,33 @@ pub fn handle_add_cookie(
         .unwrap();
 }
 
+pub fn handle_delete_cookies(
+    documents: &Documents,
+    pipeline: PipelineId,
+    reply: IpcSender<Result<(), ()>>,
+) {
+    let document = match documents.find_document(pipeline) {
+        Some(document) => document,
+        None => {
+            return reply.send(Err(())).unwrap();
+        },
+    };
+    let url = document.url();
+    reply
+        .send(
+            match document
+                .window()
+                .upcast::<GlobalScope>()
+                .resource_threads()
+                .send(DeleteCookies(url))
+                .unwrap()
+            {
+                () => Ok(()),
+            },
+        )
+        .unwrap();
```

Comment:
Let's write this as:
```rust
document
    .window()
    .upcast::<GlobalScope>()
    .resource_threads()
    .send(DeleteCookies(url))
    .unwrap();
reply.send(Ok(());
```

Reply:
A small hitch, closing parentheses in the last line
`reply.send(Ok(()));`

Reply:
@jdm I think this change made the build fail since the result can be a Err variant too. :/

Reply:
I guess you need an `unwrap` for this `send` 👀?

Reply:
We can write `let _ = reply.send(Ok(()));` to ignore the error.